azure-pipelines-full-tests.yml

```diff
@@ -13,144 +13,74 @@ pr:
       - '*'
 
 jobs:
-  - job: ExtractMetadata
-    displayName: Extract Metadata
+- job: AutomationTest
+  displayName: Automation Test (Profile Latest)
+  timeoutInMinutes: 120
+  pool:
+    vmImage: 'ubuntu-16.04'
```

Comment:
Is 16.04 too old? Consider 18.04 or 20.04

Reply:
Will update in https://github.com/Azure/azure-cli/pull/17368 to keep in sync with main pipeline.